testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -129,14 +132,9 @@ class MiniAppDisplayActivity : BaseActivity() {
                 AppSettings.instance.urlParameters
             )
         } else {
-            viewModel.obtainMiniAppDisplay(
-                this@MiniAppDisplayActivity,
-                appInfo,
-                appId!!,
-                miniAppMessageBridge,
-                miniAppNavigator,
-                AppSettings.instance.urlParameters
-            )
+            val firstTimeWindow = FirstTimeWindow(this@MiniAppDisplayActivity, this)
+            if (appInfo != null) firstTimeWindow.initiate(appInfo, "")
```

Comment:
Why do you need to check if `appInfo` is null here? Couldn't you just pass both in a single call? `firstTimeWindow.initiate(appInfo, appId!!)`

Reply:
Oh right! 
I could just simple pass in single call. I'll update this part, Thanks.